File: PooCC01CNetz/Semana2/Python/Ejercicios2/Celular.py
class Celular:
    def __init__(self):
        modelo = str
        precio = float
        activo = bool

    def calculaIVA(self):
        iva = self.precio * 0.10
        return iva

    def descuento(self, porcentDesc):
        desc = self.precio * (porcentDesc * 0.01)
        return desc

File: PooCC01CNetz/Semana2/Java/Ejercicios4/src/Celular.java
public class Celular {
    public String modelo;
    public double precio;
    public boolean activo;

    public double calculaIVA() {
        double iva = precio * 0.10;
        return iva;
    }

    public double descuento(int porcenDesc) {
        double desc = precio * (porcenDesc * 0.01);
        return desc;
    }
}
